### PR TITLE
feat: Add tencent cloud coscmd plugin

### PR DESCRIPTION
**Pull request by JohannLai**
Add tencent cloud coscmd plugin， document ：https://cloud.tencent.com/document/product/436/10976

### CODE REVIEW

File: content/orangeciplugins/tencentyun-coscmd/index.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+date: 2019-08-29T00:00:00+00:00
+title: tencentyun-coscmd
```
Comment (tboerger) [inline content/orangeciplugins/tencentyun-coscmd/index.md]: This should be a human readable title, not some kind of slug.

File: content/orangeciplugins/tencentyun-coscmd/index.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Using this tencentyun-coscmd plugin, the user can upload, download and delete the object (Object) of tencent cloud cos through simple command line instructions.
+
+Document address: https://cloud.tencent.com/document/product/436/10976
+
+```yaml
+steps:
+- name: tencentyun-coscmd
+  image: orangeciplugins/tencentyun-coscmd
+  commands:
```
Comment (tboerger) [inline content/orangeciplugins/tencentyun-coscmd/index.md]: Containers which just accept a commands block aren't Drone plugins, these are just plain docker containers.

File: content/orangeciplugins/tencentyun-coscmd/index.md (hunk 1 above)
Comment (JohannLai) [reply content/orangeciplugins/tencentyun-coscmd/index.md]: fixed

**Comment by JohannLai** [comment]
> I'm personally against adding this to the plugin index as this isn't a plugin, it's a plain Docker container.

Although this is just a simple container, I think it can be included in drone to help more people realize that the service of Tencent cloud cos can also be used in drone as plugins.  😄

**Comment by tboerger** [comment]
Users can use any docker image within drone, but plugins are wrapping specific functions which get configured via settings and they are focused on specific tasks.

If we follow your argumentation we should also add a plugin entry for the golang image to show the users they can use it ;)

**Comment by JohannLai** [comment]
ok~  I see, thanks.

**Pull request closed without merging**